### ISSUE TITLE
Conversion to g/g is in reality to mg/g

### ISSUE THREAD

**Issue by danieleongari** [yakutovicha/aiida-raspa#36]
The key `conversion_factor_molec_uc_to_gr_gr` of `output_parameters` is parsed from Raspa's output:
```
Conversion factor molecules/unit cell -> gr/gr:       1.0756975415 [-]
```
which should read instead `molecules/unit cell -> mg/g`. 
It is weird indeed that this value is on average systems close to 1.

It should be checked if this error is still present in the most recent version of Raspa (if so open an issue there) and decide how to deal with this problem, maintaining back compatibility.

**Comment by danieleongari** [comment]
Still present in RASPA v2.0.37.
Opened an issue:
https://github.com/iRASPA/RASPA2/issues/3

**Comment by yakutovicha** [comment]
fixed in #38
